src/Client/Core/Entities/CleanEntityStorage.cs

```diff
@@ -9,15 +9,26 @@ namespace Microsoft.DurableTask.Client.Entities;
 public readonly record struct CleanEntityStorageRequest
 {
     /// <summary>
-    /// Gets a value indicating whether to remove empty entities.
+    /// Gets the default request parameters. The default is meant to represent
+    /// "maximal" cleaning that is safe to call at all times.
+    /// </summary>
+    public static CleanEntityStorageRequest Default => new()
```

Comment:
You could also add a parameterless constructor and set all the bool properties to `true` there. This doesn't change `default` - but just an idea for you to consider.